run_create_appstream_stack_fleet.py

```diff
@@ -210,7 +214,7 @@ def get_subnet_and_security_group_id():
         image_name = env_s['IMAGE_NAME']
         stack_name = env_s['NAME']
         embed_host_domains = env_s['EMBED_HOST_DOMAINS']
-        desired_instances = env_s.get('DESIRED_INSTANCES', 1)
+        desired_instances = env_s.get('DESIRED_INSTANCES', 2)
```

Comment:
@GoldieKMG // 이건 2로 반드시 둬야 하는 이유가 있을까요?
```suggestion
        desired_instances = env_s.get('DESIRED_INSTANCES', 2)
```

Reply:
@yoonjs2 // 인스턴스를 띄우고 바로 확인 하기에 2로 설정하였는데 코멘트보고 곰곰히 생각해봤더니 autoscale을 더 정확히 하려면 대부분 다 default로 두는게 좋을 것 같습니다 변경해두도록 하겠습니다